openapi_server/orms/user.py

```diff
@@ -0,0 +1,18 @@
+from pynamodb.models import Model
+from pynamodb.attributes import (
+    UnicodeAttribute,
+)
+
+
+class DbUser(Model):
+    class Meta:
+        table_name: str = "User"
+        host: str = "http://host.docker.internal:4566"
+        aws_access_key_id: str = "foo"
+        aws_secret_access_key: str = "bar"
```

Comment:
We can maintain localstack.env file and put some environment variable
AWS_ACCESS_KEY_ID=XXXXXXXXX
AWS_SECRET_ACCESS_KEY=XXXXXXXXX
AWS_DEFAULT_REGION=us-west-1
They will be automatically picked up when the env variables set.